spec/features/passenger_management_spec.rb

```diff
@@ -1,12 +1,29 @@
 require 'rails_helper' 
 
 feature 'Passenger Management' do 
+  before :each do
+    sign_in(create :user, :admin)
+  end
   scenario 'creating a new passenger' do 
+    visit passengers_path
+    click_link 'Add New Passenger'
+    fill_in('passenger[name]', with: 'Foo Bar')
```

Comment:
It will give me an error saying it can't find the fields if I did 'Name' or 'Passenger name' or even just 'name'.